src/App.js

```diff
@@ -15,13 +15,39 @@ import { ScrollToTop } from "./components/atoms";
 // Routes
 import Routes from "./Routes";
 
+// Check if dark or light mode
+function getMode() {
+  let mode = localStorage.getItem("mode");
+  if (mode !== null) {
+    return JSON.parse(mode);
+  } else {
+    // Set default
+    localStorage.setItem("mode", false);
```

Comment:
Add new line because of logical separation.
Ref: https://github.com/snek-at/tonic/blob/master/STYLE_GUIDE.md